core/base/document.php

```diff
@@ -1499,4 +1523,48 @@ private function add_handle_revisions_changed_filter() {
 	private function remove_handle_revisions_changed_filter() {
 		remove_filter( 'wp_save_post_revision_post_has_changed', [ $this, 'handle_revisions_changed' ] );
 	}
+
+	private function get_runtime_data_updaters() {
+		$runtime_data_updaters = [];
+
+		foreach ( $this->data_updaters as $data_updater ) {
+			if ( $data_updater->is_update_needed() ) {
+				$runtime_data_updaters[] = $data_updater;
+			}
+		}
+
+		return $runtime_data_updaters;
+	}
+
+	private function handle_page_elements( $elements, $data_updaters, $event ) {
```

Comment:
Rename this method to `iterate_elements`

Reply:
Done.